### DISCUSSION TITLE
R Support?

### DISCUSSION THREAD

**Discussion by jam-sha** [zed-industries/zed#6627] [Feature Requests]
Hi, I was wondering if support for R is planned for the near future?

**Comment by JosephTLyons** [comment]
At the moment, it isn't planned for the near future.  We don't have many upvotes (👍) on the issue that is used to track it:

- https://github.com/zed-industries/community/issues/1040

If this number increases quite a bit, we will prioritize it, but at the moment, not many are requesting it, so it would likely wind up coming out at a community plugin once we haver a plugin system in place.

**Comment by JosephTLyons** [comment]
There is now an R-lang extension - find it by searching in the extensions store (command palette > `zed: extensions`).
